test/e2e/test/version.go

```diff
@@ -11,12 +11,14 @@ import (
 	"github.com/elastic/cloud-on-k8s/pkg/controller/common/version"
 )
 
-// Elastic Stack versions used in the E2E tests
+// Elastic Stack versions used in the E2E tests. These should be updated as new versions for each major are released.
 const (
-	// MinVersion68x minimum version for 6.8.x tested with the operator
-	MinVersion68x = "6.8.20"
-	// LatestVersion7x current latest version for 7.x
-	LatestVersion7x = "7.16.2" // version to synchronize with the latest release of the Elastic Stack
+	// LatestVersion6x is the current latest version for 6.x
+	LatestVersion6x = "6.8.20"
+	// LatestVersion7x is the current latest version for 7.x
+	LatestVersion7x = "7.17.0-SNAPSHOT"
```

Comment:
The `TestElasticMapsServerVersionUpgradeToLatest7x` test is failing because you changed the target version here to a snapshot build. So again the SNAPSHOT vs production build problem. I think you either have to change the test in `TestElasticMapsServerVersionUpgradeToLatest7x` to keep Elasticsearch on the production version as you have done already for the `TestElasticMapsServerVersionUpgradeToLatest8x` test. Or keep this variable on the latest production build.

Reply:
At least for now, `LatestVersion7x = "7.17.0-SNAPSHOT"` has to stay because `8.0.0-SNAPSHOT` already expects to be upgraded from `7.17.0` (which doesn't have a production build yet).

Reply:
Actually, the src version has to be the snapshot which doesn't mean that this variable needs to be the source. I'm making it back into latest production build and updating the comment.